tests/test_backed_sparse.py

```diff
@@ -127,6 +127,28 @@ def test_backed_indexing(
     assert_equal(csr_mem[:, var_idx].X, dense_disk[:, var_idx].X)
 
 
+@pytest.mark.parametrize(
+    "indexing_func",
+    [
+        lambda x: (Ellipsis, slice(0, 10)),
+        lambda x: (slice(0, 10), Ellipsis),
+        lambda x: (slice(0, 10), slice(0, 10), Ellipsis),
+        lambda x: (Ellipsis, slice(0, 10), slice(0, 10)),
```

Comment:
you don’t seem to use `x` in any of the cases here, also I’m adding the “ellipsis in the middle” case:

```suggestion
        (..., slice(0, 10)),
        (slice(0, 10), ...),
        (slice(0, 10), slice(0, 10), ...),
        (..., slice(0, 10), slice(0, 10)),
        (slice(0, 10), ..., slice(0, 10)),
```

or something like

```py
class Idx:
    def __class_getitem__(idx):
        return idx
```


```suggestion
        Idx[..., :10],
        Idx[:10, ...],
        Idx[:10, :10, ...],
        Idx[..., :10, :10],
        Idx[:10, ..., :10],
```